### PR TITLE
OSDOCS#6190: Update minimum support period to 16 months

### PR DESCRIPTION
**Pull request by laubai**
Version(s):
4, 12, 4.13, 4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6190

Link to docs preview:
https://64524--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-life-cycle#rosa-minor-versions_rosa-life-cycle

QE review:
- [x] QE has approved this change.
- [x] PM has approved this change

Additional information:
Not strictly asked for by this issue, but by related conversation with Nethali and PM.

### CODE REVIEW

**Comment by laubai** [comment]
In particular please check my math on the dates.

**Review by tmalove — APPROVED** [review]

File: modules/life-cycle-minor-versions.adoc [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
-. A customer's cluster is currently running on 4.8.14. The 4.8 minor version became generally
-  available on July 27, 2021.
-. On July 29, August 28, and September 12, 2022, the customer is notified that their cluster will enter "Limited Support" status
-  on September 27, 2022 if the cluster has not already been upgraded to a supported minor version.
-. The cluster must be upgraded to 4.9 or later by September 27, 2022.
+. A customer's cluster is currently running on 4.13.8. The 4.13 minor version became generally
+  available on May 17, 2023.
+. On July 19, August 16, and September 2, 2024, the customer is notified that their cluster will enter "Limited Support" status
+  on September 17, 2024 if the cluster has not already been upgraded to a supported minor version.
+. The cluster must be upgraded to 4.9 or later by September 17, 2024.
```
Comment (xueli181114) [inline modules/life-cycle-minor-versions.adoc:21]:
```suggestion
. The cluster must be upgraded to 4.14 or later by September 17, 2024.
```

**Comment by laubai** [comment]
Thanks @xueli! Made your suggested change.

**Pull request merged**